packages/wdio-local-runner/src/index.ts

```diff
@@ -16,19 +18,32 @@ export interface RunArgs extends Workers.WorkerRunPayload {
 
 export default class LocalRunner {
     workerPool: Record<string, WorkerInstance> = {}
+    private processFactory: ProcessFactory
```

Comment:
Unless we explicitly need access to it in a unit test why not just do:


```suggestion
    #processFactory = new ProcessFactory(xvfb)
```